library/build.gradle

```diff
@@ -81,6 +88,7 @@ dependencies {
     testImplementation "com.squareup.okhttp3:mockwebserver:$okhttp3Version"
     testImplementation "androidx.arch.core:core-testing:$androidXCoreVersion"
     testImplementation "com.google.truth:truth:$truthVersion"
+    testImplementation "org.robolectric:robolectric:$robolectricVersion"
```

Comment:
As I mentioned in another comment we don't need Roboelectic due to the following reasons:
1. With your changes you only need JSON, which can be added on its own.
2. Roboelectic is too big. By adding it we are seriously increasing build time for the project on the CI.
3. Roboelectic isn't maintained well enough and looks quite abandoned by Google.

Reply:
Removed robolectric dependency and used moshi for formatting json.